s4e-web/src/app/app.configuration.ts

```diff
@@ -0,0 +1,24 @@
+import {JsonObject, JsonProperty} from 'json2typescript';
+
+export interface IConfiguration {
```

Comment:
is there any convention with capital I before name?

Reply:
To be honest I don't know. I see that @michal-szostak used `IConstants` so I used `IConfiguration`.